xmind/tests/test_workbook_mixin_element.py

```diff
@@ -61,140 +58,135 @@ def test_excessive_parameters(self):
             with self.subTest(pair=pair):
                 self._test_method_by_excessive_parameters(pair, _element)
 
-        _get_attribute.assert_not_called()
-        _set_attribute.assert_not_called()
-        _getAttribute_method.assert_not_called()
-        _readable_time_method.assert_not_called()
-        _get_current_time_method.assert_not_called()
-        _setAttribute__method.assert_not_called()
-
     def test_regist_owner_workbook(self):
         """Test registOwnerWorkbook method with NOT empty ownerWorkbook object"""
         self._setOwnerDocument_method = self._init_patch_with_name(
-            '_setOwnerDocument', 'xmind.core.Element.setOwnerDocument')
+            '_setOwnerDocument', 'xmind.core.Element.setOwnerDocument', autospec=True)
         self._ownerWorkbook.getOwnerDocument.return_value = "owner"
 
         obj = WorkbookMixinElement("test", self._ownerWorkbook)
-        obj.registOwnerWorkbook()
+        self._assert_init_with_object(obj, "test")
+        self._remove_patched_function('_registOwnerWorkbook')
 
-        self.assertEqual(self._setOwnerDocument_method.call_count, 2)
+        obj.registOwnerWorkbook()
+        self._setOwnerDocument_method.assert_called_once_with(obj, "owner")
 
     def test_regist_none_owner_workbook(self):
         """Test registOwnerWorkbook method with empty ownerWorkbook object"""
         self._setOwnerDocument_method = self._init_patch_with_name(
-            '_setOwnerDocument', 'xmind.core.Element.setOwnerDocument')
+            '_setOwnerDocument', 'xmind.core.Element.setOwnerDocument', autospec=True)
         self._ownerWorkbook.getOwnerDocument.return_value = "owner"
 
-        self._ownerWorkbook = None
-        obj = WorkbookMixinElement("test", self._ownerWorkbook)
+        obj = WorkbookMixinElement("test", None)
+        self._assert_init_with_object(obj, "test")
+        self._remove_patched_function('_registOwnerWorkbook')
+
         obj.registOwnerWorkbook()
 
         self._setOwnerDocument_method.assert_not_called()
 
     def test_get_owner_workbook(self):
         """Test getOwnerWorkbook method"""
-        self._registOwnerWorkbook_method = self._init_patch_with_name(
-            '_registOwnerWorkbook', 'xmind.core.mixin.WorkbookMixinElement.registOwnerWorkbook')
         obj = WorkbookMixinElement("test", self._ownerWorkbook)
 
         self.assertEqual(obj.getOwnerWorkbook(), self._ownerWorkbook)
 
+        self._assert_init_with_object(obj, "test")
+
     def test_set_owner_workbook(self):
         """Test setOwnerWorkbook method with None owner_workbook"""
-        self._registOwnerWorkbook_method = self._init_patch_with_name(
-            '_registOwnerWorkbook', 'xmind.core.mixin.WorkbookMixinElement.registOwnerWorkbook')
         obj = WorkbookMixinElement("test", self._ownerWorkbook)
-
         obj._owner_workbook = None
 
         obj.setOwnerWorkbook(self._ownerWorkbook)
         self.assertNotEqual(None, obj._owner_workbook)
```

Comment:
I think we will get more accurate result if use self.assertEqual(self._ownerWorkbook, obj._owner_workbook) construction